spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapProperties.java

```diff
@@ -47,6 +48,11 @@
 	 */
 	private String ldif = "classpath:schema.ldif";
 
+	/**
+	 * Schema validation
+	 */
+	private Validation validation = new Validation();
```

Comment:
Nested configuration properties can be (and should be) `final`. You don't need a setter since the object instance for this properties object is always there.